wagtail/admin/templates/wagtailadmin/shared/tabs/tab_nav_link.html

```diff
@@ -11,8 +11,8 @@
 
 <a id="tab-label-{{ tab_id }}" href="#tab-{{ tab_id }}" class="w-tabs__tab {{ classname }}" role="tab" aria-selected="false" tabindex="-1">
     {{ title }}
-    <div data-tabs-errors class="w-tabs__errors">
-        <span data-tabs-errors-count aria-hidden="true"></span>
-        <span data-tabs-errors-statement class="w-sr-only"></span>
+    <div class="w-tabs__errors" data-controller="w-count" data-w-count-active-class="!w-flex" data-w-count-target-value="#tab-{{ tab_id }}" >
```

Comment:
```suggestion
    <div class="w-tabs__errors" data-controller="w-count" data-w-count-active-class="!w-flex" data-w-count-container-value="#tab-{{ tab_id }}" >
```

I think this should be the container selector (e.g. where we want to look for the target elements).